plugins/woocommerce/client/blocks/assets/js/atomic/blocks/product-elements/sale-badge/block.json

```diff
@@ -10,11 +10,20 @@
 		"isDescendentOfSingleProductTemplate": {
 			"type": "boolean",
 			"default": false
+		},
+		"isPreview": {
+			"type": "boolean",
+			"default": false
```

Comment:
Is there a specific reason we are adding the `isPreview` attribute when this is not used within the actual block? Because this is not required to show the example, you can just set `example: {}` for the preview to show:

> So even if no attributes are defined, setting an empty example object example: {} will trigger the preview to show.

^^ from the documentation: https://developer.wordpress.org/block-editor/reference-guides/block-api/block-registration/#example-optional